src/compiler/header-builder.ts

```diff
@@ -67,13 +68,13 @@ export class HeaderBuilder {
 
     res += `int ${options.prefix}_init(${options.prefix}_t* s);\n`;
     res += `int ${options.prefix}_execute(${options.prefix}_t* s, ` +
-      'const char* p, const char* endp);\n';
+    'const char* p, const char* endp);\n';
 
     res += '\n';
     res += '#ifdef __cplusplus\n';
-    res += '}  /* extern "C" *\/\n';
+    res += '}  /* extern "C" */\n';
     res += '#endif\n';
-    res += `#endif  /* ${DEFINE} *\/\n`;
+    res += `#endif  /* ${DEFINE} */\n`;
```

Comment:
what's the reason for the change of this line?

Reply:
Its an unnecessary escape character. eslint complains about it. checked it and it is true.